tidb-cloud/import-csv-files.md

```diff
@@ -85,17 +85,17 @@ To allow TiDB Cloud to access the CSV files in the Amazon S3 or GCS bucket, do o
 
 To import the CSV files to TiDB Cloud, take the following steps:
 
-1. Log in to the [TiDB Cloud console](https://tidbcloud.com/), navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page, and choose your target project on the top of the left navigation bar.
+1. Open the **Import** page in the TiDB Cloud console.
 
-2. Locate your cluster, click **...** in the upper-right corner of the cluster area, and select **Import Data**.
+    1. Log in to the [TiDB Cloud console](https://tidbcloud.com/) and navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page.
 
-3. On the **Import** page, click **Import Data** in the upper-right corner, and then select **From S3**.
+    2. If you have multiple projects, choose a target project in the left navigation pane. Otherwise, skip this step.
 
-    > **Tip:**
-    >
-    > Alternatively, you can also click the name of your target cluster on the **Clusters** page and click **Import Data** in the **Import** area.
+    3. In the row of your target cluster, click the cluster name to go to its cluster overview page, and then click **Import** in the left navigation pane.
+
+2. On the **Import** page, click **Import Data** in the upper-right corner, and then select **From S3**.
 
-4. On the **Data Import** page, provide the following information.
+3. On the **Data Import** page, provide the following information.
 
     - **Data Format**: select **CSV**.
```

Comment:
```suggestion
    - **Data format**: select **CSV**.
```

Reply:
L103:
The section is **Target database** and is not configurable
<img width="1155" alt="image" src="https://user-images.githubusercontent.com/60599231/210348313-e08827d0-4f85-4b6e-a819-b35df512b130.png">

Reply:
updated in https://github.com/pingcap/docs/pull/11883/commits/9dcfee22d99647825374af60868374dbd991d312